kitsune/products/models.py

```diff
@@ -134,6 +134,9 @@ class Topic(ModelBase):
 
     # Whether or not this topic is visible in the ui to users.
     visible = models.BooleanField(default=False)
+    # Whether or not this topic is used in the AAQ.
+    in_aaq = models.BooleanField(
+        default=False, help_text='Whether this topic is shown to users in the AAQ or not.')
```

Comment:
I don't know. If it's wrapped in gettext, it should be a lazy gettext. It could be that it grabs the msgid. I don't have any of the model help text fields in Fjord gettext'd.